packages/frontend/src/hooks/useOrganizationGroups.ts

```diff
@@ -29,7 +29,7 @@ export const useOrganizationGroups = (
 ) => {
     const setErrorResponse = useQueryError();
     return useQuery<GroupWithMembers[], ApiError>({
-        queryKey: ['organization_groups'],
+        queryKey: ['organization_groups', includeMembers, queryOptions],
```

Comment:
Nice catch 👍 Query key should always have all the variables that are part of the request. 
I think we could exclude the queryOptions though as they should have no impact on the results.

Reply:
Agreed.